content/en/_index.md

```diff
@@ -11,22 +11,19 @@ developer_note:
 
 ## High-quality, ubiquitous, and portable telemetry to enable effective observability
 
-<div class="l-primary-buttons mt-5">
+<div class="l-primary-buttons">
```

Comment:
IMHO, you should reinstate the `mt-5`, since it given balance (equal space) before and after the description.